### PR TITLE
AC Automated Test LRAC-8421 | master

### PR DESCRIPTION
**Pull request by LucasAcacio**
https://issues.liferay.com/browse/LRAC-8421
https://issues.liferay.com/browse/LRAC-8813 (refactor)

### CODE REVIEW

**Comment by LucasAcacio** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase [hunk 1]
```diff
@@ -300,12 +300,91 @@ definition {
...
+				fieldName = "Text");
+
+			FormFields.editText(
+				fieldName = "Text",
+				fieldValue = "Test Assets");
 
-		// TODO LRAC-8421 CanTimeFilterUseCustomRangeInFormsBehaviourReport
+			FormPortlet.submitLocalizedSuccessfully(buttonName = "Submit");
+
+			Pause(locator1 = "5000");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase:330]: Hi @LucasAcacio, I think this **pause** is not necessary, it is only necessary to put this pause if you include one `for`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase [hunk 2]
```diff
@@ -300,12 +300,91 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8421 | Test Summary: Assert Assets Forms Custom Range Time Filter in Behaviour Report	"
-	@ignore = "true"
 	@priority = "5"
 	test CanTimeFilterUseCustomRangeInFormsBehaviourReport {
+		property portal.upstream = "quarantine";
+		// AC Refactor: Timing issues may happen on this test case
+		// AC Refactor ticket: LRAC-8813
+
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase:309]: Hi @LucasAcacio, I think you forgot to do a connection with DXP

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase [hunk 3]
```diff
@@ -300,12 +300,91 @@ definition {
...
+			ACCards.viewMetricValue(
+				cardTitle = "Visitors Behavior",
+				metricName = "Views",
+				metricValue = "${expected value}");
+
+			ACCards.viewMetricValue(
+				cardTitle = "Visitors Behavior",
+				metricName = "Completion Time",
+				metricValue = "${expected value}");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase]: Hi @LucasAcacio, what do you think of putting this macro in a `for`?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase (hunk 1 above)
Comment (LucasAcacio) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase:330]: ok

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase (hunk 3 above)
Comment (LucasAcacio) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase]: The expected different kind of values there, but I'll change the variable names so people can understand that

**Comment by LucasAcacio** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @LucasAcacio , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Pull request closed without merging**